vispy/geometry/parametric.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014, Nicolas P. Rougier
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+
+import numpy as np
+from .normals import normals
```

Comment:
Whoops. The surface function uses normals(), and I haven't tested that it works with the existing normal implementation. It's not possible to put back normals.py and merge the two implementations later once we properly test, compare, and benchmark them?